clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -90,11 +91,16 @@ def describe_clusters():
             cluster_value += "  The cluster is stable.\n"
         else:
             cluster_value += "  The cluster is *not* stable.\n"
+            exit_value = 1
 
         if len(cluster) != 0:
             for node, state in cluster.iteritems():
                 cluster_value += "    {} is in state {}.\n".format(node, state)
 
             print cluster_value
 
+    # This makes sure an error value is returned when the clusters are not
+    # stable
+    sys.exit(exit_value)
```

Comment:
We've discussed how your shouldn't call `sys.exit` in a function - it makes it hard to reuse. Instead, you should return from the function, then use `sys.exit` at the top level.